idlak-misc/cart_lts/run.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# Takes a lexicon a generates letter-to-sound rules
+
+if [ $# -ne 2 ]
+    then
+        echo "Usage is "$0" <input lexicon> <output file>"
+        exit 1
+    else
+        lexicon=$1
+        output=$2
+fi
+
+if [[ ${lexicon: -4} != ".lex" ]]
+    then
+        echo "Lexicon must be a .lex file"
+        exit 1
+fi
+
+align="${lexicon::-4}"".align"
+converted="converted.lex"
+
+mkdir -p tmp
+phonet2cartltspath="../../idlak-misc/cart_lts/tmp/"
```

Comment:
normally we leave the / off the end of the path and put it in where it is used, makes it a bit more clear

Reply:
check to see if it has been set then default it to this value (makes it more configurable)